skimage/util/_flood_fill.py

```diff
@@ -0,0 +1,180 @@
+"""flood_fill.py - inplace flood fill algorithm
+
+This module provides a function to fill all equal (or within tolerance) values
+connected to a given seed point with a different value.
+"""
+
+import numpy as np
+
+
+def flood_fill(image, seed_point, new_value, *, selem=None, connectivity=None,
+               indices=False, tolerance=None, inplace=False):
```

Comment:
Don't mean to railroad you but both @stefanv and myself feel like util is not the right place for this... I suggest having it in morphology and cloning it to segmentation. 

Long term, we could consider a "Special" group for things that are more user-facing than util but still generic.

I guess the question I would ask is, if a user is crawling through the segmentation docs, do we want them to stumble upon this function?

Reply:
An alternative would be to add entries to the right `See Also` sections in segementation.
(In case you missed it, there was already an earlier [discussion](https://github.com/scikit-image/scikit-image/pull/3245#discussion_r202028213) about this.)